en/3-transaction/transactions.md

```diff
@@ -0,0 +1,257 @@
+
+Example:
+
+```json
+{
+   "attributes":[
+      {
+         "usage":48,
+         "data":"cf62fd54fc761f291d07d68088dd81b8b35a7c444f3af8acd78a3ad4ff75d163"
+      },
+      {
+         "usage":48,
+         "data":"aac6d49da8f63cf6442c5f707317bc3e7490029af1a75b83adc0ec3b1b3e1f0f"
+      },
+      {
+         "usage":240,
+         "data":"e5a487e6b3a8e58a9fe883bde7acace4b880e6aca1e6b58be8af95efbc8ce880b6efbc81202048656c6c6f2072656d61726b73efbc81"
+      },
+      {
+         "usage":144,
+         "data":"5473685f323031372f322f382031363a31383a353931373033323132343035"
+      }
+   ]
+}
+```
+
+#### Attributes usage types
+The following usage types can be included in the transaction attributes.
+
+| Usage type    | Name              | Description                           | Length                                  |
+|---------------|-------------------|---------------------------------------|-----------------------------------------|
+| `0`           | `ContractHash`    | Script hash of the contract           | `32`                                    |
+| `2`           | `ECDH02`          | Public key for ECDH key exchange      | `32`                                    |
+| `3`           | `ECDH03`          | Public key for ECDH key exchange      | `32`                                    |
+| `32`          | `Script`          | Additional validation of transactions | `20`                                    |
+| `48`          | `Vote`            | Voting payload                        | Should be specified (up to 255 bytes)   |
+| `129`         | `DescriptionUrl`  | URL for description                   | Should be specified (up to 255 bytes)   |
+| `144`         | `Description`     | Description                           | Should be specified (up to 255 bytes)   |
+| `161` - `175` | `Hash1-Hash15`    | Storing of custom hash values         | `32`                                    |
+| `240` - `255` | `Remark-Remark15` | Custom general remarks                | Should be specified (up to 65535 bytes) |
+
+A maximum of 16 attributes can be added to each transaction.
+
+### Outputs
+An output `vout` is an amount of an asset transferred to an address as a result of the transaction. Each transaction can have up to `65536` outputs. It depends on the type of the transaction how the outputs are validated.
+
+| Type                        | Description                                                                                   |
+|-----------------------------|-----------------------------------------------------------------------------------------------|
+| `IssueTransaction`          | Requires the transaction to be signed by the asset administrative contract                    |
+| `ClaimTransaction`          | Requires one or more unclaimed outputs to be referenced in the transaction `claims` attribute |
+
+All other transactions require one or more unspent outputs to be referenced in the transaction `inputs` attribute.
```

Comment:
MinerTransaction requires 0 inputs, too?

Reply:
This has been updated in the code now. Thanks for noticing :+1: